Dockerfile

```diff
@@ -0,0 +1,37 @@
+# choose the Image which has Node installed already
+FROM node:16 AS build
+
+# add arg for the network
+ARG NETWORK
```

Comment:
please add another `ARG VERSION` for weblets' version

Reply:
done.